easybuild/tools/options.py

```diff
@@ -2011,6 +2012,14 @@ def opts_dict_to_eb_opts(args_dict):
             prefix = '--'
         option = prefix + str(arg)
         value = args_dict[arg]
+
+        if str(arg) in allow_multiple_calls:
+            if not isinstance(value, (list, tuple)):
+                value = [value]
+            for v in value:
+                args.append(option + '=' + str(v))
+            continue
```

Comment:
I think some people would not be happy to break out of the loop in the middle like this. What about instead leaving the list intact below for the `allow_multiple_calls` special cases, and then adding a new `elif` clause at the end of the loop to handle this new case

Reply:
Umh it would still be cleaner to check for `allow_multiple_calls` first and than check if the value is a list.
If we want to avoid `continue` I think the other 2 `if` can become `elif` as all the value checked after are not compatible anyway

Reply:
Yeah, that was also my original thinking, move the 
```
        if isinstance(value, (list, tuple)):
            value = ','.join(str(x) for x in value)
```
into the main if clause and deal with the differences there

Reply:
Should be fixed with https://github.com/easybuilders/easybuild-framework/pull/4667/commits/550d96cb8aeeab15644d995d8dc9f4c1ca5f549f